src/coralogix.js

```diff
@@ -104,7 +104,19 @@ class CoralogixLogger extends FormattedLoggerBase {
     });
   }
 
-  _logImpl(payload, fields) {
+  async _logImpl(payload, fields) {
```

Comment:
I don't think this should be changed into async w/o changing the logger base. after all, the caller doesn't expect it to be async.

Reply:
It always was async in that it returned a promise, so just declaring the method async doesn't actually change anything.
Additionally, all callers (e.g. MultiLoggers) are designed to handle async Loggers.